agent/bench-scripts/tests/pbench-trafficgen/test-51.txt

```diff
@@ -219,35 +219,35 @@ Iteration 4-profile2-0.001pct_drop complete (4 of 4), with 1 pass and 0 failures
-/var/tmp/pbench-test-bench/opt/pbench-agent/bench-scripts/postprocess/trafficgen-postprocess /var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-51_tg:trex-profile_pf:profile1,profile2_ml:0.002,0.001_tt:bs_1900.01.01T00.00.00/2-profile1-0.001pct_drop/sample2 testhost.example.com none   profile1 unknown
-/var/tmp/pbench-test-bench/opt/pbench-agent/bench-scripts/postprocess/trafficgen-postprocess /var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-51_tg:trex-profile_pf:profile1,profile2_ml:0.002,0.001_tt:bs_1900.01.01T00.00.00/2-profile1-0.001pct_drop/sample3 testhost.example.com none   profile1 unknown
-/var/tmp/pbench-test-bench/opt/pbench-agent/bench-scripts/postprocess/trafficgen-postprocess /var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-51_tg:trex-profile_pf:profile1,profile2_ml:0.002,0.001_tt:bs_1900.01.01T00.00.00/3-profile2-0.002pct_drop/sample1 testhost.example.com none   profile2 unknown
-/var/tmp/pbench-test-bench/opt/pbench-agent/bench-scripts/postprocess/trafficgen-postprocess /var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-51_tg:trex-profile_pf:profile1,profile2_ml:0.002,0.001_tt:bs_1900.01.01T00.00.00/3-profile2-0.002pct_drop/sample2 testhost.example.com none   profile2 unknown
-/var/tmp/pbench-test-bench/opt/pbench-agent/bench-scripts/postprocess/trafficgen-postprocess /var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-51_tg:trex-profile_pf:profile1,profile2_ml:0.002,0.001_tt:bs_1900.01.01T00.00.00/3-profile2-0.002pct_drop/sample3 testhost.example.com none   profile2 unknown
-/var/tmp/pbench-test-bench/opt/pbench-agent/bench-scripts/postprocess/trafficgen-postprocess /var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-51_tg:trex-profile_pf:profile1,profile2_ml:0.002,0.001_tt:bs_1900.01.01T00.00.00/4-profile2-0.001pct_drop/sample1 testhost.example.com none   profile2 unknown
-/var/tmp/pbench-test-bench/opt/pbench-agent/bench-scripts/postprocess/trafficgen-postprocess /var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-51_tg:trex-profile_pf:profile1,profile2_ml:0.002,0.001_tt:bs_1900.01.01T00.00.00/4-profile2-0.001pct_drop/sample2 testhost.example.com none   profile2 unknown
-/var/tmp/pbench-test-bench/opt/pbench-agent/bench-scripts/postprocess/trafficgen-postprocess /var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-51_tg:trex-profile_pf:profile1,profile2_ml:0.002,0.001_tt:bs_1900.01.01T00.00.00/4-profile2-0.001pct_drop/sample3 testhost.example.com none   profile2 unknown
+/var/tmp/pbench-test-bench/opt/pbench-agent/bench-scripts/postprocess/trafficgen-postprocess /var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-51_tg:trex-profile_pf:profile1,profile2_ml:0.002,0.001_tt:bs_1900.01.01T00.00.00/1-profile1-0.002pct_drop/sample1 testhost.example.com    profile1 unknown
```

Comment:
Yes, that is why the dumping of the `.cmd` files was ported from `b0.69` to here in a separate commit before the changes to `pbench-trafficgen` are made.

Reply:
In `bash` the 7 arguments are still there, the test infrastructure capture of the arguments to `trafficgen-postprocess` is not very good (see https://github.com/distributed-system-analysis/pbench/blob/main/agent/bench-scripts/test-bin/mock-postprocess/mock-cmd#L3).

Reply:
It may not be "good", but it is nice and "simple" -- a few minutes spent noodling and I could not come up with an acceptable alternative.  (I mean, sure, changing `$*` to `$@` would help, but then we still need to add a bunch of superstructure to deal with quoting each individual argument, and I'm thinking that that's not a win, given that we have the `.cmd` file.)